src/ml-service/packages/sensor_location/locate/requirements.txt

```diff
@@ -0,0 +1,24 @@
+click==7.1.1
+dnspython==1.16.0
```

Comment:
For some reason, this dependency always requests me to reinstall it every time I test out the latest changes. Kinda strange.